bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx

```diff
@@ -145,14 +170,20 @@ class AudioModal extends Component {
     const {
       inputDeviceId,
       outputDeviceId,
+      skipCheck,
     } = this.props;
 
+
+    if (skipCheck) {
```

Comment:
Instead of checking inside the `handleGoToEchoTest` you should only call this function when you want the echoTest state.

`skipCheck ? handleGoToEchoTest : handleJoinMicrophone`